app/decorators/person_decorator.rb

```diff
@@ -2,11 +2,11 @@ class PersonDecorator < Draper::Decorator
   delegate_all
 
   def twitter_url
-    "http://twitter.com/#{twitter}" if twitter
```

Comment:
Hi @KaraAJC  can you change this pull request to use .blank? instead of `.nil? || .empty?`